src/sdp4.h

```diff
@@ -54,7 +54,8 @@ struct _sdp4 {
 	stepn, step2, preep, pl, sghs, xli, d2201, d2211, sghl, sh1, pinc,
 	pe, shs, zsingl, zcosgl, zsinhl, zcoshl, zsinil, zcosil;
 
-
+	//Variables that are used locally in SGP4(), but also are used to calculate squint angle elsewhere
```

Comment:
I don''t understand this comment. We only calculate squint angle for SDP4, not SGP4. Is it a typo, or is it something I don't get?